### DISCUSSION TITLE
Can contributing code to the project receive airdrops

### DISCUSSION THREAD

**Discussion by imalasong** [Titannet-dao/Community#6] [General]
https://github.com/Titannet-dao/titan-node

**Comment by MaryTitan** [comment]
Absolutely! Please refer to our[ RetroPGF Program ](https://github.com/Titannet-dao/Community/discussions/3). We will distribute credits to those who contribute to us! Thank you for your interest!

**Reply by imalasong** [reply]
nice

**Reply by gudduurr** [reply]
Good

**Comment by eddielueng** [comment]
好的

**Comment by Amran3321** [comment]
TNT3

**Comment by Amran3321** [comment]
![Screenshot_2025-06-15-20-23-18-92](https://github.com/user-attachments/assets/42c22ef4-081a-4a7b-ab30-9a50365f9ecf)
![Screenshot_2025-06-15-20-23-12-30](https://github.com/user-attachments/assets/510c9636-4a5d-437e-a400-b0dcb3ab2045)

**Comment by Amran3321** [comment]
Still valid Points TNT 3
